src/modules/phone/index.ts

```diff
@@ -20,10 +20,39 @@ export class PhoneModule extends ModuleBase {
    *
    * @since 7.3.0
    */
-  number(): string {
-    const format = this.faker.helpers.arrayElement(
-      this.faker.definitions.phone_number.formats
-    );
+  number(): string;
+
+  /**
+   * Generates a random phone number.
+   *
+   * @param options Options object
+   * @param options.style Style of the phone number. Defaults to 'human'
+   *
+   * @see faker.string.numeric(): For generating a random string of numbers.
+   * @see faker.helpers.fromRegExp(): For generating a phone number matching a regular expression.
+   *
+   * @example
+   * faker.phone.number() // '961-770-7727'
+   *
+   * @since 7.3.0
+   */
+  number(options?: { style: 'human' | 'national' | 'raw' }): string;
+  number(options?: {
+    /**
+     * Style of the generated phone number:
+     * - `'human'`: (default) A human-input phone number, e.g. 555-770-7727 or 555.770.7727 x1234
```

Comment:
```suggestion
     * - `'human'`: (default) A human-input phone number, e.g. `555-770-7727` or `555.770.7727 x1234`
```

Same for the others.